cpp/src/parquet/arrow/path_internal_test.cc

```diff
@@ -113,9 +114,50 @@ class MultipathLevelBuilderTest : public testing::Test {
       ArrowWriteContext(default_memory_pool(), arrow_properties_.get());
 };
 
-TEST_F(MultipathLevelBuilderTest, NonNullableSingleListNonNullableEntries) {
+class MultipathLevelListTool {
```

Comment:
@emkornfield I've change the original `list` test to `list` and `list_view` test, would you think it's ok?